sdk/internal/test/tracing/validator_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tracing
```

Comment:
Would be good to use the `_test` suffix here so that you have to import `/internal/test/tracing` (or whatever we end up calling it).